addons/binding/org.openhab.binding.paradoxalarm/ESH-INF/thing/zone.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="paradoxalarm"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="zone">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ip150" />
+		</supported-bridge-type-refs>
+		<label>Paradox zone</label>
+		<description>Paradox zone</description>
+
+		<channels>
+			<channel id="label" typeId="zoneLabel" />
+			<channel id="isOpened" typeId="isOpenedState" />
+			<channel id="isTampered" typeId="isTamperedState" />
+			<channel id="hasLowBattery" typeId="hasLowBatteryState" />
+		</channels>
+
+		<config-description>
+			<parameter name="id" type="integer" min="1" max="192" required="true">
+				<label>Zone Id</label>
+				<description>This is the zone ID of the Paradox zone</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="zoneLabel">
+		<item-type>String</item-type>
+		<label>Zone label</label>
+		<description>Label of zone</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+	<channel-type id="isOpenedState">
+		<item-type>contact</item-type>
+		<label>Zone state</label>
+		<description>State of zone</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+	<channel-type id="isTamperedState">
+		<item-type>boolean</item-type>
```

Comment:
Hm... I'm not sure about this. Boolean represents best these states because it says true - is tampered, or false - not tampered. 
While Contact will permit easier graphical presentation of these states (green/red for ex.) I'm not sure if it's natural to bind them to OPEN/CLOSED state. Maybe we should use String here. What do you think?